modules/cram/templates/concat_vcf.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -euo pipefail
+
+concat () {
```

Comment:
https://samtools.github.io/bcftools/bcftools.html#concat
>In other words, the default behavior of the program is similar to unix "cat" in that when two files contain a record with the same position, that position will appear twice on output.

Is this the desired behavior? Might want to use  `--remove-duplicates`? I don't understand what 'ligation' is, might be of interest to this concat command as well?

Reply:
This part of the docs makes me think we do not want to use --ligate
"Note that the option is intended for VCFs with perfect overlap, sites in overlapping regions present in one but missing in 
the other are dropped"